docker/Dockerfile.debug

```diff
@@ -11,10 +11,11 @@ RUN apt-get update && \
     net-tools \
     libc6-dbg gdb valgrind \
     vim \
-    nano \
-    emacs \
-    wrk
```

Comment:
you can drop emacs.. but please do keep nano. I am vi handicapped. Second, do we need another tier of debug images? May be sometime in future, but I was under the impression that our debug is kinda kitchen sink, and not a production friendly image. For production, this would be super stripped down for both perf and security. We would not be running valgrind, nor would we have vim/emacs/nano/fortio or anything, except envoy, agent and iptables script